### ISSUE TITLE
Markdown editor is not accessible with screenreader

### ISSUE THREAD

**Issue by heull001** [haiwen/seahub#5479]
The markdown-editor on Webclient is completely inaccessible with screenreader.

All buttons are not labeled, so screenreader-users are not able to find the right button, there is no ability to format text or at least to save changes.

The  plain text editor for markdown also has the problem, that the textfield is not recognized as such. Screenreader-Users are not able to focus on it and so are not able to make changes.

I don't know how complicated it is to fix this, as a workaround for the plain markdown editor it could be possible to offer a button to switch to the normal editor for txt-files, this works fine.

This problems occure on windows 10 with lates Firefox and Edge as browsers and lates JAWS and NVDA as screenreaders and also on lates iOS with safari as browser and voiceover as screenreader, so I think, it is a general problem.

For testing on Windows, I would recommend to get NVDA from https://www.nvaccess.org/download/
